docs/source/user/installing.rst

```diff
@@ -260,7 +260,9 @@ information.
 +----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
 | Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
 +===========+==============+===========================+============================+==============================+===================+=============================+
-| 0.57.0    | TBC          | 3.8.x <= version < 3.12   | 1.21 <= version < 1.25     | 0.40.x                       | 11.x              | 2021.x                      |
+| 0.57.0    | TBC          | 3.8.x <= version < 3.12   | 1.21 <= version < 1.25     | 0.40.x                       | 14.x              | 2021.x                      |
+|           |              |                           |                            |                              | (11.x for Linux   |                             |
```

Comment:
Do we still plan to go with only requiring 11.x, assuming the llvmdev PR I make today resolves the LLVM 14 issues to everyone's satisfaction?

Reply:
Oops, just saw @sklam's comment on the main PR!

Reply:
Now that GOT patch is merged, we can drop the part about llvm 11